cluster/src/main/java/org/apache/iotdb/cluster/client/sync/SyncDataClient.java

```diff
@@ -39,13 +39,20 @@
   public SyncDataClient(TProtocolFactory protocolFactory, Node node, SyncClientPool pool)
       throws TTransportException {
     // the difference of the two clients lies in the port
-    super(protocolFactory.getProtocol(new TFastFramedTransport(new TSocket(node.getIp(),
-        node.getDataPort(), RaftServer.getConnectionTimeoutInMS()))));
+    super(protocolFactory.getProtocol(new TimeoutChangeableTFastFramedTransport(
+        new TSocket(node.getIp(), node.getDataPort(), RaftServer.getConnectionTimeoutInMS()))));
     this.node = node;
     this.pool = pool;
     getInputProtocol().getTransport().open();
   }
 
+  public void setTimeout(int timeout) {
+    ((TimeoutChangeableTFastFramedTransport) (getInputProtocol().getTransport()))
+        .setTimeout(timeout);
+    ((TimeoutChangeableTFastFramedTransport) (getOutputProtocol().getTransport()))
+        .setTimeout(timeout);
```

Comment:
The same transport is used in both input and output, so it is unnecessary to set twice.

Reply:
Indeed. I will remove one.